Packs/CortexXpanse/Playbooks/Xpanse_-_Alert_Handler.yml

```diff
@@ -241,20 +241,210 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "7":
+    continueonerrortype: ""
+    form:
+      description: Add one or multiple tags to the assets(s) associated with the current XSOAR incident.
+      expired: false
+      questions:
+      - defaultrows: []
+        fieldassociated: ""
+        gridcolumns: []
+        id: "0"
+        label: ""
+        labelarg:
+          simple: What tag(s) do you want to add to asset(s)?
+        options: []
+        optionsarg: []
+        placeholder: ""
+        readonly: false
+        required: false
+        tooltip: specificy what tags you want to add to the asset(s) associated to the current XSOAR incident in comma separated value format
+        type: shortText
+      sender: Your SOC team
+      title: Determine tags to add to asset(s)
+      totalanswers: 0
+    id: "7"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    message:
+      bcc:
+      body:
+      cc:
+      format: ""
+      methods: []
+      subject:
+      timings:
+        completeafterreplies: 1
+        completeaftersla: false
+        completeafterv2: true
+        retriescount: 2
```

Comment:
Why are retries needed for a data collection task?

Reply:
weird that this seems to be the default at least for XSOAR8 (maybe because it default to email collection task and keeps the retry even if you remove).  can remove this

Reply:
fixed in [John feedback](https://github.com/demisto/content/pull/35111/commits/d007fb518328c2e1d2cf4beb5cf1bc6e66c40819)